Software/Python/gopigo.py

```diff
@@ -43,6 +43,24 @@
 import struct
 import subprocess
 
+# Test GoPiGo vs GoPiGo3
+# If
+try:
+    import gopigo3
+    try:
+        GPG3 = gopigo3.GoPiGo3()
+    except:
+        print("This library (gopigo.py) is for the GoPiGo1 and GoPiGo2.")
+        print("No GoPiGo3 detected.")
+        print("WARNING:  If you are using the GoPiGo3 hardware this software will not work!")
+        print("WARNING:  If you are using the GoPiGo3 hardware this software will not work!")
+        time.sleep(5)
+except:
+    print("This library is for the GoPiGo1 and GoPiGo2.")
+    print("No GoPiGo3 Library Detected.")
+    print("WARNING:  If you are using the GoPiGo3 hardware this software will not work!")
+    time.sleep(5)
```

Comment:
Do you think we want to slow them down if the there's no `gopigo3` module installed or if the communication with the `gopigo3` fails?

Reply:
I wouldn't print anything at all if they're using the gopigo library with a gopigo2 (or 1). That's a perfectly valid use, and having something printed automatically is 1. scary 2. annoying.